.vscode/extensions.json

```diff
@@ -0,0 +1,7 @@
+{
+  "recommendations": [
+    "msjsdiag.debugger-for-chrome",
+    "dbaeumer.vscode-eslint"
+  ],
+  "unwantedRecommendations": ["esbenp.prettier-vscode"]
```

Comment:
This is the most important line in this file. Users are being actively discouraged from using this extension in codebases where eslint is used as the default formatter.